### PR TITLE
Sanic performance options documentation

### PR DESCRIPTION
**Pull request by szokeasaurusrex**
## Pre-merge checklist

*If you work at Sentry, you're able to merge your own PR without review, but please don't unless there's a good reason.*

- [x] Checked Vercel preview for correctness, including links
- [x] PR was reviewed and approved by any necessary SMEs
- [x] PR was reviewed and approved by a member of the [Sentry docs team](https://github.com/orgs/getsentry/teams/docs)

## Description of changes

Added documentation of the `unsampled_statuses` option, which we have added in [sentry-python #2491](https://github.com/getsentry/sentry-python/pull/2419).

- [Sentry Docs contributor guide](https://docs.sentry.io/contributing/)

### CODE REVIEW

**Review by sentrivana — APPROVED** [review]

File: src/platforms/python/common/integrations/sanic/index.mdx [hunk 1]
```diff
@@ -41,6 +41,33 @@ sentry_sdk.init(
 app = Sanic(__name__)
 ```
 
+### Configuration options
+
+If you are using performance monitoring, you can configure the Sanic integration to not send transactions for requests which have resulted in certain HTTP statuses using the `unsampled_statuses` option.
```
Comment (shanamatthews) [inline src/platforms/python/common/integrations/sanic/index.mdx:46]:
```suggestion
If you're using performance monitoring, you can configure the Sanic integration to not send transactions for requests that resulted in certain HTTP statuses using the `unsampled_statuses` option.
```

File: src/platforms/python/common/integrations/sanic/index.mdx [hunk 2]
```diff
@@ -41,6 +41,33 @@ sentry_sdk.init(
...
+
+By default, the Sanic integration generates transactions for all requests, except requests which result in a 404 HTTP status. If you wish to generate transactions for all HTTP statuses, including 404, or change the HTTP statuses which have no transactions generated, you can manually instantiate the Sanic integration when initializing the Sentry SDK, like so:
```
Comment (shanamatthews) [inline src/platforms/python/common/integrations/sanic/index.mdx:48]:
```suggestion
By default, the Sanic integration generates transactions for all requests, except requests that result in a 404 HTTP status. To generate transactions for all HTTP statuses, including 404, or change which HTTP statuses don't generate transactions, you can manually instantiate the Sanic integration when initializing the Sentry SDK, like so:
```

File: src/platforms/python/common/integrations/sanic/index.mdx [hunk 3]
```diff
@@ -41,6 +41,33 @@ sentry_sdk.init(
...
+    # including 404
+    integrations=[SanicIntegration(unsampled_statuses=None)]
+
+    # Set traces_sample_rate to 1.0 to capture 100%
+    # of transactions for performance monitoring.
+    traces_sample_rate=1.0,
+)
+```
+
+If you wish to specify a different set of HTTP statuses for which no transactions should be generated, set `unsampled_statuses` to a set containing the HTTP statuses which should have no transactions generated. For example, if you wish to have no transactions for requests resulting in a 200 or 404 HTTP status, use `SanicIntegration(unsampled_statuses={200, 400})`.
```
Comment (shanamatthews) [inline src/platforms/python/common/integrations/sanic/index.mdx:69]:
```suggestion
To specify a different set of HTTP statuses for which no transactions should be generated, set `unsampled_statuses` to a set containing the HTTP statuses that don't generate transactions. For example, if you wish to have no transactions for requests resulting in a 200 or 404 HTTP status, use `SanicIntegration(unsampled_statuses={200, 400})`.
```

**Review by shanamatthews — APPROVED** [review]
Just a few language tweaks!

**Pull request merged**